templates/pages/links.html

```diff
@@ -4,10 +4,16 @@
 
 {% block content%}
 
-	<div>
-		<p>We gather useful links to tutorials, good practices, etc. </p>
-
+    <div align="center" class="naslov">
+		<p><b>Useful links to design ideas, tutorials, good practices and more: </b></p>
 	</div>
-
+    <div align="center">
```

Comment:
We're using CSS, so you can make a new class for this list of links (hint: check out the [`<ul>`tag](http://www.w3schools.com/tags/tag_ul.asp)) and have alignment (and other cool stuff!) defined in the CSS. The cool part about defining things like that in CSS is that you can reuse the same style in multiple places and change it just in one place when you think of something new :)